dolphin/db/sqlalchemy/models.py

```diff
@@ -121,3 +121,20 @@ class Disk(BASE, DolphinBase):
     model = Column(String(255))
     media_type = Column(String(255))
     capacity = Column(Integer)
+
+
+class AlertSource(BASE, DolphinBase):
+    """Represents an alert source configuration."""
+    __tablename__ = 'alert_source'
+    storage_id = Column(String(36), primary_key=True)
+    host = Column(String(255))
+    version = Column(String(255))
+    community_string = Column(String(255))
+    username = Column(String(255))
+    password = Column(String(255))
```

Comment:
what is the difference between password and encryption_password

Reply:
Password is for authentication, while encryption_password is for encryption, it's about USM for SNMPv3.

Reply:
Got it thanks